lesson_13/dragla_marius/lesson13_inheritance.py

```diff
@@ -0,0 +1,86 @@
+class PerimeterMixin:
+
+    def perimeter(self):
+        return sum(self.sides)
+
+class Polygons(PerimeterMixin):
```

Comment:
Placing the Mixin here will make the perimeter available for all shapes that inherit from Polygon.

Were we can move the mixin to have the perimeter method only on instances of Triangles?

Reply:
moved the Mixin to Triangles class